src/resources/authoring/gv2.json

```diff
@@ -1,67 +1,60 @@
-[
-  [
-    [
-      {
-        "comment": "*** Level 1, Mission 1.1: Playground ***",
-        "template": "GenomePlayground",
-        "initialDrake": {
-          "alleles": "M-M, T-T, w-w, C-C, B-B, D-D, rh-rh, Bog-Bog",
-          "sex": 1
-        },
-        "userChangeableGenes": "wings, forelimbs, hindlimbs, armor, horns"
-      }
-    ],
-  ],
-  [
-    [
-      {
-        "comment": "*** Level 2, Mission 1.1: Genome Challenge (basics intro, user drake visible) ***",
-        "template": "GenomeChallenge",
-        "instructions": "Match the target drake!",
-        "showUserDrake": true,
-        "randomizeTrials": true,
-        "initialDrake": {
-          "alleles": "Bog-, D-, rh-rh",
-          "sex": 1
-        },
-        "targetDrakes": [{
-          "alleles": "",
-          "sex": 0
-        },
-        {
-          "alleles": "",
-          "sex": 1
-        }],
-        "userChangeableGenes": "wings, forelimbs, hindlimbs",
-        "linkedGenes": {
-          "drakes": [0, 1],
-          "genes": "metallic, tail, horns, color, black, dilute, armor, nose, bogbreath"
-        }
+{
+  "definitions": {
```

Comment:
Definitions seems a little vague to me, personally. Definitions of what? How about just "challenges"? Or is the worry that that name may not be correct in the future?